### PR TITLE
Added Complete NodeJS + ExpressJS + MongoDB Course by Complete Coding by Prashant Sir to free-courses-hi.md

### PR DESCRIPTION
**Pull request by HRIDYANSHU054**
## What does this PR do?
Add Complete NodeJS + ExpressJS + MongoDB  🚀🔥 Course ❤️ in Hindi | Notes | Certification by Complete Coding by Prashant Sir to free-courses-hi.md

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -511,6 +511,7 @@
 
 #### Node.js
 
+* [Complete NodeJS + ExpressJS + MongoDB  🚀🔥 Course ❤️ in Hindi | Notes | Certification](https://www.youtube.com/playlist?list=PL78RhpUUKSwfeSOOwfE9x6l5jTjn5LbY3) - Complete Coding by Prashant Sir
```
Comment (eshellman) [inline courses/free-courses-hi.md:514]: no emojis

Comment (HRIDYANSHU054) [reply courses/free-courses-hi.md:514]: Done

**Comment by HRIDYANSHU054** [comment]
@eshellman I've removed the emojis. Could you please check if it's ready to be merged?

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -511,6 +511,7 @@
 
 #### Node.js
 
+* [Complete NodeJS + ExpressJS + MongoDB Course in Hindi | Notes | Certification](https://www.youtube.com/playlist?list=PL78RhpUUKSwfeSOOwfE9x6l5jTjn5LbY3) - Complete Coding by Prashant Sir
```
Comment (eshellman) [inline courses/free-courses-hi.md:514]:
```suggestion
* [Complete NodeJS + ExpressJS + MongoDB Course in Hindi \| Notes \| Certification](https://www.youtube.com/playlist?list=PL78RhpUUKSwfeSOOwfE9x6l5jTjn5LbY3) - Complete Coding by Prashant Sir
```
Escape pipes

**Pull request merged**